ui/fisher/seabase.js

```diff
@@ -23,28 +23,26 @@ class SeaBase {
   }
 
   getConnection() {
-    let _this = this;
+    return new Promise((resolve, reject) => {
+      let req = window.indexedDB.open(this._dbName, this._dbVersion);
```

Comment:
I think if the variable won't change in the future, declare it as `const` would be better than `let` ?
`let` here and some ones below can be changed.

Reply:
I don't disagree, but this pattern is used everywhere. I think I'd rather clean this up in a separate pull request.